Don't ignore webpack errors

At the moment any bunding errors are ignored by the webpack task.

Here's an [example project](https://github.com/nigredo-tori/various-examples/tree/master/scala/sbt-scalajs-bundler/webpack-bail). It contains an error (an unresolved module). But the `webpack` task succeeds!
```
$ sbt fastOptJS::webpack
<truncated>
[info] ERROR in ./js-resources-fastopt.js
[info] Module not found: Error: Cannot resolve module 'no-such-module' in /home/dmitry/work/various-examples/scala/sbt-scalajs-bundler/webpack-bail/target/scala-2.11/scalajs-bundler/main
[info]  @ ./js-resources-fastopt.js 1029:32-57
[success] Total time: 19 s, completed Jan 13, 2017 3:46:32 PM
```

The reason for this is a quirk of webpack itself: by default it always produces zero return code, even when errors are encountered. There are ways to change this behaviour, though: `--bail` command-line option and `bail` flag in configuration. I think scalajs-bundler should also expose this functionality, and/or fail on error by default.

Workaround: Setting `bail` config flag in a custom webpack config.

--- a/sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala
+++ b/sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala
@@ -122,7 +122,7 @@ object Webpack {
     */
   def run(args: String*)(workingDir: File, log: Logger): Unit = {
     val webpackBin = workingDir / "node_modules" / "webpack" / "bin" / "webpack"
-    val cmd = "node" +: webpackBin.absolutePath +: args
+    val cmd = "node" +: webpackBin.absolutePath +: "--bail" +: args
     Commands.run(cmd, workingDir, log)
     ()
   }

